### ISSUE TITLE
NM Presentation Titles [2075 Batch]

### ISSUE THREAD

**Issue by ErSKS** [KCE/NM#8]
Dear Students, select the title from following list & comment your choice with team details, [FIFO Selection]:

- [x] Applications of NM in Science & Engineering
- [x] Taylor Series & Its Applications
- [x] Finite Element Methods
- [x] Gradient Descent Methods
- [ ] Monte Carlo Method
- [ ] Error Propagation & Analysis
- [ ] Cubic Spline
- [x] Fast Fourier Transform
- [x] Curve Fitting
- [ ] Continuous Optimization
- [x] Markov Decision Process
- [x] Hyperparameters Optimization
- [x] Romberg's Method
- [x] Runge-Kutta Methods
- [x] Relaxation Methods

**Presentation Guidelines:**

- 3/4 members per group
- 15 Mins. for Presentation
- Slide Count: 20 - 30
- Cover Followings:
     - Introduction
     - Technical/Numerical Aspects
     - Pros & Cons
     - Corresponding Applications
     - Include Graphics (Images, Vectors, Animations, etc.)
     - Conclusion

_Note:- You can search topics for your group & need to finalize the topics before preparation._

**Comment by anupkafle** [comment]
TOPIC :Applications of NM in Science & Engineering

TEAM MEMBERS:
Anup Kafle KCE075BCT011
Anushil Timsina KCE075BCT012
Sakar poudel kCE075BCT037
Sandeep subedi KCE075BCT039

**Comment by JuJu2181** [comment]
Topic : Gradient Descent Methods 
Team Members:
Anish Shilpakar (KCE075BCT008)
Anjaan Khadka (KCE075BCT009)
Rubin Baidhya(KCE075BCT034)
Sairush Tamang(KCE075BCT036)

**Comment by Aaryashree03** [comment]
Topic : Runge-Kutta Methods
Team Members:
Aaryashree Pokharel (KCE075BCT002)
Nikita Pokharel(KCE075BCT021)
Riya Parajuli(KCE075BCT029)
Shyamron Dongol(KCE075BCT043)

**Comment by prashant5126** [comment]
Topic : Hyperparameters Optimization
Team Members:
Prashant Manandhar --KCE075BCT023
Pratik Tamrakar -- KCE075BCT025
Roman Shilpakar--KCE075BCT031
Nitesh Kumar Das--KCE075BCT022

**Comment by sendess** [comment]
Topic : Markov Decision Process

Team Members:
Manjil Karki----------KCE075BCT019
Pratik Shakya--------KCE075BCT024
Ravi Pandit ----------KCE075BCT028
Sandesh Acharya----KCE075BCT040

**Comment by Anumhrzn** [comment]
Topic: Romberg's Method

Team Members:
Anu Maharjan(KCE075BCT010)
Esha Valandyo(KCE075BCT016)
Laxmi Shrestha(KCE075BCT018)
Rakshya Belbase(KCE075BCT027)

**Comment by Amitduwal11** [comment]
Topic: Monte Carlo Method

Team Members:
Aabhas Dhaubanja (KCE075BCT001)
Amit Duwal (KCE075BCT006)
Anish Machamasi (KCE075BCT007)
Ronish Shrestha (KCE075BCT032)

**Comment by iamrosan** [comment]
Topic : Fast Fourier Transform
Members:
Juhel Phanju (075BCT017)
Roshan Shrestha (075BCT033)
Sachin Manandhar (075BCT035)
Sudip Manandhar (075BCT045)

**Comment by twayanaaayush** [comment]
Topic::
Taylor Series & it's Application

Presenters::
Aayush Twayana (075BCT003)
Sameep Gaihre (075BCT038)
Sanish Shrestha (075BCT041)
Sanjog Gaihre (075BCT042)

**Comment by RochakSedai** [comment]
Topic::
Curve Fitting

Presenters::
Abhinandan Shrestha(075BCT004)
Rochak Sedai(075BCT030)
Sudip Shrestha(075BCT046)
Umesh Mahatto(075BCT048)

**Comment by dpesjoc29** [comment]
TOPIC :Finite Element Method
TEAM MEMBERS:

Bibek Manandhar KCE075BCT013
Dipesh Joshi KCE075BCT015
Pratirodh Sharma Pokharel kCE075BCT026
Sushant Tha  KCE075BCT043

**Comment by NabinKawan** [comment]
Topic:  Continuous Optimization
Team Members:-
  -Nabin Kawan (KCE075BCT020)
  -Sudeep Kasichhwa (KCE075BCT044)
  -Bimal Bhoydhyo (KCE075BCT014)
  -Aditya Jaishi (KCE075BCT05)
